cockatrice/src/deck_loader.cpp

```diff
@@ -200,9 +200,19 @@ void DeckLoader::saveToStream_DeckZoneCards(QTextStream &out, const InnerDecklis
 QString DeckLoader::getCardZoneFromName(QString cardName, QString currentZoneName)
 {
     CardInfo *card = db->getCard(cardName);
-    if(card && card->getIsToken())
+    if (card && card->getIsToken())
         return DECK_ZONE_TOKENS;
 
     return currentZoneName;
 }
 
+QString DeckLoader::getCompleteCardName(QString cardName)
```

Comment:
To further enforce in the method signature your intentions, its best to change to `QString DeckLoader::getCompleteCardName(const QString cardName) const`